co2calculator/calculate.py

```diff
@@ -71,7 +74,7 @@ def calc_co2_car(
         )
     # Check if distance of stops provided
     if distance is None and stops is None:
-        assert ValueError(
+        raise ValueError(
```

Comment:
@han16nah, Important: This is a change to the business logic! I saw this method's pattern of complaining differentiating from the others. This is why I changed from `assert` to `raise` the error.